python/ray/tests/kuberay/scripts/scale_up.py

```diff
@@ -1,19 +1,25 @@
 import ray
-from ray._private import test_utils
+from ray._private.test_utils import wait_for_condition
 
 
-@test_utils.wait_for_stdout(
```

Comment:
I changed it to use the cluster resource as testing explicitly rather than depending on the autoscaler print outs (which is not really source of truth for cluster scaled up e2e): 
i.e. the log string ('Adding ...') could still be present when the actual scaling fails (i.e. fails to launch the pod)